test/wh_test_crypto.c

```diff
@@ -87,16 +88,254 @@ static int _cancelCb(uint16_t seq)
 }
 #endif
 
+
+#ifdef HAVE_ECC
+static int whTest_CryptoEcc(whClientContext* ctx, WC_RNG* rng)
+{
+    int ret = WH_ERROR_OK;
+    ecc_key eccPrivate[1];
+    ecc_key eccPublic[1];
+#define TEST_ECC_KEYSIZE 32
+    uint8_t shared_ab[TEST_ECC_KEYSIZE] = {0};
+    uint8_t shared_ba[TEST_ECC_KEYSIZE] = {0};
+    uint8_t hash[TEST_ECC_KEYSIZE] = {0};
+    uint8_t sig[ECC_MAX_SIG_SIZE] = {0};
+
+#if 0
+    whNvmFlags flags = WH_NVM_FLAGS_NONE;
+    whKeyId key_id_a = WH_KEYID_ERASED;
+    uint8_t* label_a = (uint8_t*)("Ecc Label A");
+    whKeyId key_id_b = 24;
+    uint8_t* label_b = (uint8_t*)("Ecc Label B");
+#endif
```

Comment:
dead code

Reply:
I'll add in the additional key testing styles, just like curve25519.  This won't be dead then.